packages/design-system/src/components/primitives/dnd/use-drop.ts

```diff
@@ -72,9 +68,6 @@ type UseDropProps<Data> = {
   // |___________|
   emulatePointerAlwaysInRootBounds?: boolean;
 
-  // Distance from an edge when placement is put next to an element edge
-  placementPadding?: number;
```

Comment:
I am missing it, how did you replace this setting?

Reply:
It is moved to computeIndicatorPlacement. Only there is needed.